### PR TITLE
Add sample for updating a cluster in Dataproc.

### PR DESCRIPTION
**Pull request by sudeepag**

### CODE REVIEW

**Comment by sudeepag** [comment]
Signed the CLA.

File: dataproc/create_cluster_and_submit_job.py [hunk 1]
```diff
@@ -119,6 +119,35 @@ def get_cluster_id_by_name(cluster_list, cluster_name):
     cluster = [c for c in cluster_list if c['clusterName'] == cluster_name][0]
     return cluster['clusterUuid'], cluster['config']['configBucket']
 
+# [START update_cluster]
+def update_cluster(dataproc, project, cluster_name, zone, num_workers):
+    """Updates the specified cluster to the given number of workers.
+    Use 'config.worker_config.num_instances' and 'workerConfig' for workers.
+    Use 'config.secondary_worker_config.num_instances' and
+    'secondaryWorkerConfig' for preemptible workers."""
+    print('Updating cluster to %d workers.' % num_workers)
```
Comment (waprin) [inline dataproc/create_cluster_and_submit_job.py]: We use {} and format() rather then % for string interpolation in this repo

File: dataproc/create_cluster_and_submit_job.py [hunk 2]
```diff
@@ -119,6 +119,35 @@ def get_cluster_id_by_name(cluster_list, cluster_name):
     cluster = [c for c in cluster_list if c['clusterName'] == cluster_name][0]
     return cluster['clusterUuid'], cluster['config']['configBucket']
 
+# [START update_cluster]
```
Comment (waprin) [inline dataproc/create_cluster_and_submit_job.py]: no need for a region tag unless you're including it from a doc

**Comment by waprin** [comment]
Thanks for the PR. I think it'd be better to a) move this into a totally separate file with a main function b) write an integration test that verifies the changes to the resource was made.

**Comment by waprin** [comment]
closing based on lack of author response, feel free to re-open.

**Pull request closed without merging**